reference/5.1/Microsoft.PowerShell.Core/About/about_Jobs.md

```diff
@@ -240,33 +229,26 @@ Get-Job
 The results appear in a table. The status of the job appears in the **State**
 column.
 
-```
+```Output
 Id Name  PSJobTypeName State    HasMoreData Location  Command
 -- ----  ------------- -----    ----------- --------  -------
 1  Job1  BackgroundJob Complete True        localhost Get-Process
 2  Job2  BackgroundJob Running  True        localhost Get-EventLog -Log ...
 3  Job3  BackgroundJob Complete True        localhost dir -Path C:\* -Re...
 ```
 
-In this case, the State property reveals that Job 2 is still running. If you
-were to use the `Receive-Job` cmdlet to get the job results now, the results
-would be incomplete. You can use the `Receive-Job` cmdlet repeatedly to get all
-of the results. By default, each time you use it, you get only the results that
-were not already received, but you can use the **Keep** parameter of the
-`Receive-Job` cmdlet to retain the results, even though they were already
-received.
+In this case, the **State** property reveals that Job 2 is still running. If
+you were to use the `Receive-Job` cmdlet to get the job results now, the
+results would be incomplete. You can use the `Receive-Job` cmdlet repeatedly to
+get all of the results. Use the **State** property to determine when the job is
+complete.
 
-You can write the partial results to a file and then append newer results as
-they arrive or you can wait and check the state of the job later.
-
-You can use the **Wait** parameter of the `Receive-Job` cmdlet, which does not
-return the command prompt until the job is complete and all results are
-available.
+You can also use the **Wait** parameter of the `Receive-Job` cmdlet. When use
+use this parameter, the cmdlet does not return the command prompt until the job
+is complete and all results are available.
```

Comment:
I'd change complete to "completed" since the intent is that the job is past tense before **Wait** returns.

Reply:
fixed